thirdparty/README.md

```diff
@@ -35,6 +35,24 @@ The Consent oracle stores the relationship between a `consentId` and an `fspId`.
 
 ## Deploying the 3P-API
 
+Ensure that Backend Dependencies are deployed. Refer to the [example-mojaloop-backend](../example-mojaloop-backend/README.md) Helm chart for instructions details.
```

Comment:
I think there is a typo for the TTK tests  ml-ttk-test-val-tk.tests.enabled=true 
Also what is the full helm command to enable the ttk tests when deploying the thirdparty chart separately to the main mojaloop chart.

Reply:
@tdaly61, would you be a bit more specific here...

I.e.what exactly is the typo?